packages/kbn-es-query/src/kuery/node_types/__tests__/function.js

```diff
@@ -73,6 +73,14 @@ describe('kuery node types', function () {
         expect(_.isEqual(expected, result)).to.be(true);
       });
 
+      it('should accept and pass on an optional config paramter to the returned function type\'s ES query representation', function () {
```

Comment:
Although this test does confirm the config object is getting passed along by using a date field in the test query, I think it still might be better to simply use a spy here.

Reply:
Done